chunky/src/java/se/llbit/chunky/ui/controller/ChunkyFxController.java

```diff
@@ -374,6 +373,7 @@ public void exportMapView() {
           Dialogs.stayOnTop(loadWorldConfirm);
           if (loadWorldConfirm.showAndWait().orElse(ButtonType.CANCEL) == ButtonType.YES) {
             mapLoader.loadWorld(newWorld.getWorldDirectory());
+            getChunkSelection().setSelection(chunky.getSceneManager().getScene().getChunks());
```

Comment:
If the world is already loaded, we shhould also update the selection. Otherwise, do we already clear the selection?

Reply:
Good point, and no we don't clear the selection currently